core/hosting/HostWithCoreClrHost/src/SampleHost.cpp

```diff
@@ -0,0 +1,324 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <string>
+
+// https://github.com/dotnet/coreclr/blob/master/src/coreclr/hosts/inc/coreclrhost.h
+#include "coreclrhost.h"
+
+#define MANAGED_ASSEMBLY "ManagedLibrary.dll"
+
+// Define OS-specific items like the CoreCLR library's name and path elements
+#if WINDOWS
```

Comment:
It would be nice to move the WINDOWS vs. LINUX stuff to separate OS specific files.  (With a common interface)

Reply:
This is a sample so I would personally prefer a single file rather than a more complex project with additional files.

Reply:
Yea, I tried to keep everything in a single file for the sake of the sample. A more real-world host would want to split stuff out.